src/Models/ElementContent.php

```diff
@@ -41,6 +41,11 @@ public function getSummary()
         return DBField::create_field('HTMLText', $this->HTML)->Summary(20);
     }
 
+    public function getContent()
+    {
+        return DBField::create_field('HTMLText', $this->HTML)->Summary(20);
```

Comment:
I think it makes sense to use `getSummary` here instead, and refer to it as `ContentSummary` or something in the JS - it might get confusing referring to it as content because you're just taking a summary :wink:

Reply:
Great idea. This is pretty similar to how `FormField::getSchemaData` communicates an arbitrary array of data from PHP to React form field components, then they're in charge of dealing with the data that their PHP counterparts provides. We could then combine `getIconClass` into this too